app/src/main/res/values/strings.xml

```diff
@@ -121,6 +121,11 @@
     <string name="settings_key_disable_lockscreen_while_viewing_card" translatable="false">pref_disable_lockscreen_while_viewing_card</string>
     <string name="settings_key_allow_content_provider_read" translatable="false">pref_allow_content_provider_read</string>
     <string name="settings_key_oled_dark" translatable="false">pref_oled_dark</string>
+
+    <string name="settings_key_use_volume_keys" translatable="false">use_volume_keys_navigation</string>
+    <string name="volume_keys_navigation" translatable="false">Use volume keys to switch visible card</string>
+    <string name="volume_keys_navigation_summary" translatable="false">Use the volume up/down buttons to navigate between saved cards.</string>
+
```

Comment:
Only the key should be untranslatable :)

And I'd also like a bit more consistent naming so it's more clear these variables belong together.

```suggestion
    <string name="settings_key_use_volume_key_navigation" translatable="false">use_volume_key_navigation</string>
    <string name="volume_key_navigation">Use volume keys to switch visible card</string>
    <string name="volume_key_navigation_summary">Use the volume up/down buttons to navigate between saved cards.</string>
```

Reply:
Updated the untranslatable attribute and used already existing naming convention for these new settings as well.